### DISCUSSION TITLE
How To Connect Virtual Host to cloudflared?

### DISCUSSION THREAD

**Discussion by jauhari** [leokhoa/laragon#833] [Q&A]
I have Local PC that already Installed Laragon and setup Virtual Host, I was made this Virtual Host Name server.demos.net and I can access it via Local Browser normally. Then I tried bring this Virtual Host online via **CloudFlared**. We already have domain (example jakatarub.com) and I want Public Internet when Access my JakaTarub.com goes to my Virtual Host (server.demos.net)

I was setup all **cloudflared** without any issues, the problem is. When I tried access my JakaTarub.com the content goes to Default Laragon not goes to my Virtual Host. Any clues what wrong? and how to fix it?

**Comment by phoringit** [comment]
You can do this with CloudFlared Zero Trust tunnel. You need to configure your tunnel 

1. Go to Zero Trust dashboard. access your tunnel under network. if you do not have any tunnel create one and or if you have already created one simply click on three dot and go to configure 
2. Under Public hostnames click Add public hostname
3. Select your domain from your list
4. select service type
5. enter your local url (server.demos.net)
